projects/storefrontstyles/scss/cxbase/blocks/link.scss

```diff
@@ -30,3 +30,16 @@ a {
     text-decoration: none;
   }
 }
+
+.cx-static-action-link {
```

Comment:
@suprishi, please, never do styling like this.
There is a corresponding styling file for quote-links: `_quote-links.scss`